examples/integration/test/fixtures/bwb.xcodeproj/project.pbxproj

```diff
@@ -11912,6 +11906,7 @@
 				DEPLOYMENT_LOCATION = "$(INDEXING_DEPLOYMENT_LOCATION__$(INDEX_ENABLE_BUILD_ARENA)),";
 				DSTROOT = "$(PROJECT_TEMP_DIR)";
 				ENABLE_DEFAULT_SEARCH_PATHS = NO;
+				GCC_OPTIMIZATION_LEVEL = 0;
```

Comment:
`clang` default is `-O0`, but Xcode default is `-Os`, so we need to override this to allow the CC params files to work correctly.